README.md

```diff
@@ -39,15 +40,15 @@ server**. In fact, if you wish, you can use Uniqush to setup a service similar t
 - A: We are trying to make it easier. `uniqush-push` provides RESTful APIs. In
   other words, you talk with `uniqush-push` through HTTP protocol. As long as
 there's an HTTP client library for your language, you can use it and talk with
-`uniqush-push`. For details about the our RESTful APIs, you may want to check
-out our [document](http://uniqush.org/documentation/index.html).
+`uniqush-push`. For details about the our RESTful APIs, see [our API
+documentation](http://uniqush.org/documentation/index.html).
```

Comment:
The uniqush documentation is at index.html.

The documentation of the POST requests, etc. of the REST API are http://uniqush.org/documentation/usage.html